src/gurt/dlog.c

```diff
@@ -376,6 +549,7 @@ void d_vlog(int flags, const char *fmt, va_list ap)
 	 * errno to its original value
 	 */
 	int save_errno = errno;
+	int rc;
```

Comment:
Not sure what happened to my comment so I'm making it again.  Below, we do clog_lock and then proceed to do gettimeofday.  I think we should do the gettimeofday outside of the locked region to minimize the exclusive period.   Also, we might consider using something other than gettimeofday such as the gurt time routines that use clock_gettime instead as I believe gettimeofday is a system call.   Particularly if we are mostly avoiding system calls except on buffer flushes.

Reply:
Well, to me both clock_gettime() and gettimeofday() should be similar and not of a high cost since they are likely to be handled in vdso code on most architectures.

Reply:
@jolivier23 do you agree with my previous comment ? Also last CI run is reported as having failures, but all tests seem to have been successful....